### PR TITLE
SALTO-5023: Do not allow Activating Flow of type InvalidDraft

### PR DESCRIPTION
**Pull request by AlmogMesilaty**
SALTO-5023: Do not allow Activating Flow of type InvalidDraft

---
_Additional context for reviewer_:
image

---
_None_ 

---
_None_

### CODE REVIEW

File: packages/salesforce-adapter/src/change_validators/flows.ts [hunk 1]
```diff
@@ -263,6 +270,9 @@ const activeFlowValidator =
       .filter(isActivatingChange)
       .map(change => {
         if (isActivatingChangeOnly(change)) {
+          if (getFlowStatus(change.data.before) === INVALID_DRAFT) {
```
Comment (Uri-Bar) [inline packages/salesforce-adapter/src/change_validators/flows.ts:273]: are you sure you want to throw this error only if its activating change only? 
I think that we want to error this even if the user did other changes, so this needs to be outside this if

**Review by Uri-Bar — COMMENTED** [review]
looks good, just one important comment

File: packages/salesforce-adapter/src/change_validators/flows.ts (hunk 1 above)
Comment (AlmogMesilaty) [reply packages/salesforce-adapter/src/change_validators/flows.ts]: I agree, done.

**Review by Uri-Bar — APPROVED** [review]
🔥

**Pull request merged**